examples/main/views/geodata-creator-example.js

```diff
@@ -1,136 +0,0 @@
-// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
This is the example that tried to use the draggable-points-overlay. Do we need to provide a working example for that?